ui/src/version/deployment/components/StandardTransformerForm.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright 2020 The Merlin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import {
+  EuiAccordion,
+  EuiButton,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiSpacer
+} from "@elastic/eui";
+import { FeastTransformationPanel } from "./FeastTransformationPanel";
+import {
+  Config,
+  FeastConfig,
+  newConfig
+} from "../../../services/transformer/TransformerConfig";
+import { feastEndpoints, useFeastApi } from "../../../hooks/useFeastApi";
+
+const yaml = require("js-yaml");
+
+export const StandardTransformerForm = ({ transformer, onChange }) => {
+  const [config, setConfig] = useState({});
+
+  const [configInitialized, setConfigInitialized] = useState(false);
+  useEffect(() => {
+    if (configInitialized) {
+      return;
+    }
+
+    if (transformer.env_vars && transformer.env_vars.length > 0) {
+      const envVar = transformer.env_vars.find(
+        e => e.name === "TRANSFORMER_CONFIG"
```

Comment:
Can you make `TRANSFORMER_CONFIG` a constant? Also the value should be `STANDARD_TRANSFORMER_CONFIG`